ui/src/app/machines/views/MachineDetails/MachineNetwork/NetworkTable/NetworkTable.tsx

```diff
@@ -64,6 +71,21 @@ const generateRows = (
     const numaNodes = getInterfaceNumaNodes(machine, nic);
     const vlan = vlans.find(({ id }) => id === nic.vlan_id);
     const fabric = vlan ? fabrics.find(({ id }) => id === vlan.fabric) : null;
+    const discoveredSubnetId =
+      nic?.discovered?.length && nic.discovered.length > 0
```

Comment:
Is the extra `&&` necessary? Would just `nic?.discovered?.length > 0` work?

Reply:
It's a strict mode requirement, otherwise you're possibly checking `undefined > 0` which isn't an allowed comparison.